README.md

```diff
@@ -1,58 +1,42 @@
 # Micromasters
 Portal for learners and course teams to access MITx Micromasters programs
 
-## Getting Started
+## Major Dependencies
+- Python 3
```

Comment:
This is only installed inside Docker, we shouldn't list it as a requirement here